easybuild/easyconfigs/PnetCDF-1.12.3-gompi-2022b.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'ConfigureMake'
```

Comment:
Please move this into the correct directory `easybuild/easyconfigs/p/PnetCDF/PnetCDF-1.12.3-gompi-2022b.eb`

Reply:
Oh dear, stupid, mistake, will change this. Thanks